files/ansible/module_utils/test_result_parser/test_models/testmodel.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import os
+import re
+
+try:
+    from test_models.parsing_strategy import ParsingStrategy
+except ImportError:
+    from ansible.module_utils.test_result_parser.test_models.parsing_strategy import ParsingStrategy
+
+class TestModel(object):
+    def __init__(self):
+        self.fail_msg = "Default Failure Message"
+        self.parameters_regex = dict()
+        self.result_regex = re.compile(r'', re.M)
+        self.result_list = []
+        self.failure_test = lambda results, threshold : False
+        self.strategy = ParsingStrategy()
+
+    def failed(self, parsed_results, test_threshold):
```

Comment:
I'm taking into account the fact that the parsing might require more than a lambda, for future test results models that might get added to the module.

Reply:
It may also be the case that the lambda will get more complicated instead. We don't know now and there's no point in assuming either way. Simple code is self explanatory, too many layers of indirection make for confused reading and debugging.

ie. Let the decision on whether to create a dispatch function or not for when we really need it.